packages/sandpack-core/src/resolver/utils/pkg-json.ts

```diff
@@ -48,8 +49,11 @@ export function processPackageJSON(
     }
   }
 
+  let hasExports = false;
+
   // load exports if it's not the root pkg.json
   if (content.exports && pkgRoot !== '/') {
```

Comment:
Doesn't seem like it ignores non-exports in this case though? It still processes all other things in the package.json. Would be great to hear some more about this, I can help out with getting this PR over the finish line.

Reply:
Yeah, this makes sense and you are right. What I propose:
- move this block to the top of the function since `exports` always have the top priority + return early with the `exports`-based aliases if there are any. We also need to be cautious about `package.json#exports: false` - it is also a valid value. We probably should just return an empty `aliases` object in such a case or something.
- if there is no `exports` field then we should continue reading from the other fields

How does it sound?

Reply:
Pushed out the change. No extra tests added though - it's hard to craft on the spot one that would fail with the previous shape of the code

Reply:
Awesome thanks, could you alsk update the jest snapshot using yarn test -u so we can get this merged?

Reply:
Done